src/MTGCore/API/DeckController.cs

```diff
@@ -29,27 +29,24 @@ public DeckController(IRepoContext context, MTGService mtgservice, IMapper mappe
         }
 
         //api/deck/
-        [Route("Add/{deckID}/{multiverseID}")]
+        [Route("Add/{deckID}/{cardID}")]
         [HttpPost]
-        public async Task Post(int deckID, int multiverseID)
+        public async Task Post(int deckID, string cardID)
         {
```

Comment:
Is there any reason why the cardID is a string? Does the API return it as a string?

Reply:
not sure, will try with int and see if it works